Demo/Common/Networking/SessionRequest.swift

```diff
@@ -44,8 +44,6 @@ internal struct SessionRequest: APIRequest {
         
         try container.encode(installmentOptions, forKey: .installmentOptions)
         
-        // store payment method required fields
-        try container.encode("askForConsent", forKey: .storePaymentMethodMode)
```

Comment:
I assume this field in the example caused an error due to api version being below 70 :)